engine/src/main/java/com/datatorrent/stram/webapp/OperatorDiscoverer.java

```diff
@@ -167,17 +179,21 @@ else if (qName.equalsIgnoreCase("method")) {
       }
     }
 
-    private void addTagToMethod(Map<String, MethodInfo> methods, String tagText, boolean isDescription)
+    private void addTagToMethod(Map<String, MethodInfo> methods, String tagText, MethodTagType tagType)
```

Comment:
From the method where this is invoked, I was planning to use MethodTagType.valueOf(String tag) however we are not capturing all types of tags, for eg. @return @param are ignored and they are not in MethodTagType so there will be an if condition nevertheless that checks if it is one of the MethodTagType that OperatorDiscoverer cares about. 
Since that needs to be done, the tag type is known there itself and that is why we can pass it to this method. This avoids re-finding the enum in this method.

I didn't quite understand your comment but hopefully the above explanation will be sufficient.